lib/builders/flow.js

```diff
@@ -4,6 +4,9 @@ const { AbstractJsBuilder } = require('./abstract-js')
 const { ARRAY_OF_LITERAL } = require('./base')
 /*:: const { Union, Interface, Method, Field } = require('../store')*/
 
+function methodNameToPayloadName(methodName) {
+  return `${methodName[0].toUpperCase()}${methodName.slice(1)}Payload`
```

Comment:
Maybe use package change-case?

Reply:
I thought that this was simple enough to not add another dependency. But I can add one, if you want to :)